mne_bids_pipeline/_config.py

```diff
@@ -1,12 +1,13 @@
 # Default settings for data processing and analysis.
 
-from typing import Annotated, Any, Callable, Literal, Optional, Sequence, Union
+from collections.abc import Sequence
+from typing import Annotated, Any, Callable, Literal, Optional, Union
 
 from annotated_types import Ge, Interval, Len
 from mne import Covariance
 from mne_bids import BIDSPath
 
-from mne_bids_pipeline.typing import (
+from mne_bids_pipeline._types import (
```

Comment:
Speaking of public vs private, this one was meant to be public like [numpy.typing](https://numpy.org/devdocs/reference/typing.html) because people might want to annotate with some of our types like ArbitraryContrast

Reply:
Like `from mne_bids_pipeline.typing import ArbitraryContrast` or similar

Reply:
I will revisit this tomrrow

Reply:
@larsoner I have reverted this change